src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2624,21 +2605,27 @@ namespace Akka.Dispatch.SysMsg
         public Akka.Actor.IActorRef Child { get; }
         public override string ToString() { }
     }
-    public sealed class Supervise : Akka.Actor.INoSerializationVerificationNeeded, Akka.Dispatch.SysMsg.ISystemMessage
+    public sealed class Supervise : Akka.Dispatch.SysMsg.SystemMessage
     {
         public Supervise(Akka.Actor.IActorRef child, bool async) { }
         public bool Async { get; }
         public Akka.Actor.IActorRef Child { get; }
         public override string ToString() { }
     }
-    public sealed class Suspend : Akka.Actor.INoSerializationVerificationNeeded, Akka.Dispatch.SysMsg.ISystemMessage
+    public sealed class Suspend : Akka.Dispatch.SysMsg.SystemMessage, Akka.Dispatch.SysMsg.IStashWhenWaitingForChildren
     {
-        public static Akka.Dispatch.SysMsg.Suspend Instance { get; }
+        public Suspend() { }
         public override string ToString() { }
     }
-    public sealed class Terminate : Akka.Actor.INoSerializationVerificationNeeded, Akka.Actor.IPossiblyHarmful, Akka.Dispatch.SysMsg.ISystemMessage
+    public abstract class SystemMessage : Akka.Actor.INoSerializationVerificationNeeded, Akka.Dispatch.SysMsg.ISystemMessage
+    {
+        protected SystemMessage() { }
+        public bool Unlinked { get; }
+        public void Unlink() { }
+    }
+    public sealed class Terminate : Akka.Dispatch.SysMsg.SystemMessage, Akka.Actor.IPossiblyHarmful
     {
-        public static Akka.Dispatch.SysMsg.Terminate Instance { get; }
+        public Terminate() { }
```

Comment:
`Terminate` is no longer a singleton. No system message instance can ever be shared between two actors, due to the linked lists built into them now.